### ISSUE TITLE
CMS/Framework sometimes 'crashes' (javascript)

### ISSUE THREAD

**Issue by JorisDebonnet** [silverstripe/silverstripe-framework#5889]
Unfortunately I don't have a reproducable scenario to trigger this bug, and it's also hard to pinpoint what's going wrong (partly because it happens in "combinedfiles"). 

In any case, it always happens when I click on another page in the page structure, the loading icon shows, and it never stops (so then I check the console and see the error). 

In Firefox 50, this is the error I see. Looks like it occurs when it tries to retrieve  `elem.contentWindow.document` somewhere (but that's in jQuery core). 

```
Error: Permission denied to access property "document"
lib.js:354:859
.contents
assets/_combinedfiles/lib.js:354:859
.map
assets/_combinedfiles/lib.js:43:283
jQuery.fn[name]
assets/_combinedfiles/lib.js:354:1044
.onremove/<
assets/_combinedfiles/leftandmain.js:56:285
.each
assets/_combinedfiles/lib.js:37:788
jQuery.prototype.each
assets/_combinedfiles/lib.js:14:50
.onremove
assets/_combinedfiles/leftandmain.js:56:251
$.entwine.Namespace<.one/one
assets/_combinedfiles/lib.js:1633:375
.build_addrem_proxy/<
assets/_combinedfiles/lib.js:1695:56
[...]
```

**Comment by JorisDebonnet** [comment]
I guess I shouldn't be looking in lib.js, but leftandmain.js instead. I followed that issue to LeftAndMain.ActionTabSet.js, the `onremove` function. So something in here, the `.contents()` I assume, causes jQuery to crash: 
```
var frame = $('.cms-container').find('iframe');
frame.each(function(index, iframe){
	$(iframe).contents().off('click.ss-ui-action-tabset');
});
```

**Comment by JorisDebonnet** [comment]
I found that my issue is related to Translatable -- although it should probably still be solved here. 

It happens when:
- I have an `iframe` inside the page Content on the original language
- I create a translation of this page
- I visit that translation, which shows "Original content:" including the iframe, under the translated Content
- I navigate to another page in the site tree ==> CRASH! ('loading' icon displayed forever)

So the action of navigating to a new page must be doing something with iframes. I found above that it targets all `$('.cms-container').find('iframe')`, probably assuming that it will only target TinyMCE iframes that way, and not thinking about iframes that a module or other Extension might have added.
